### DISCUSSION TITLE
Milvus Error when using collection search

### DISCUSSION THREAD

**Discussion by HoangSang1510** [milvus-io/milvus#35332] [Q&A and General discussion]
Hello, i am using milvus 2.4.0, when i load collection and using collection query, it work normally but when i using collection search, i got this error

[2024-08-07 10:53:27,109] WARNING in decorators: [search] retry:9, cost: 3.00s, reason: <_InactiveRpcError: StatusCode.UNAVAILABLE, failed to connect to all addresses; last error: UNKNOWN: ipv4:172.16.200.46:19530: Failed to connect to remote host: Connection refused>
[2024-08-07 10:53:30,126] ERROR in decorators: RPC error: [search], <MilvusException: (code=101, message=failed to search: collection not loaded[collection=451674119090946895])>, <Time:{'RPC start': '2024-08-07 10:53:17.306848', 'RPC error': '2024-08-07 10:53:30.126018'}>

here is my code:

```
from config.config import *
from pymilvus import (
    connections, utility, Collection, FieldSchema, CollectionSchema, DataType)

connections.connect("default", host=MILVUS_HOST, port=MILVUS_PORT)
collection = Collection(COLLECTION_NAME)
collection.load()
list_collection = utility.list_collections()


res_query = collection.query(
    expr="id > 1", 
    output_fields=['id'], 
    timeout=5.0
)
# query working normally 

img = cv2.imread('image_test/test.png')
embedding_vec = feature_extraction(img)
search_params = {"metric_type": "COSINE", "params": {"nprobe": 10}}
res_search = collection.search(
    data=[embedding_vec],
    anns_field="vector",
    param=search_params,
    expr="active in [1]",
    output_fields=['id'],
    limit=TOP_K)
# error when search
```

**Comment by xiaofan-luan** [comment]
Could you please refer this [doc](https://github.com/milvus-io/milvus/tree/master/deployments/export-log) to export the whole Milvus logs for investigation? For Milvus installed with docker-compose, you can use docker-compose logs > milvus.log to export the logs.

**Reply by HoangSang1510** [reply]
Here is my log
[milvus.log](https://github.com/user-attachments/files/16519558/milvus.log)

**Reply by HoangSang1510** [reply]
Some WARN logs I received
![Screen Shot 2024-08-07 at 3 25 05 PM](https://github.com/user-attachments/assets/a77592f2-a1af-4076-84fb-8287958581ba)
![Screen Shot 2024-08-07 at 3 27 20 PM](https://github.com/user-attachments/assets/e81320ca-ad67-4317-b7d1-a3b5f15ac20b)

**Comment by xiaofan-luan** [comment]
And we recommend to deploy milvus latest 2.4.7
